src/mad_spark_alt/core/fast_orchestrator.py

```
@@ -0,0 +1,246 @@
+                
+                result.conclusion = await conclusion_synthesizer.synthesize_conclusion(
+                    problem_statement=problem_statement,
+                    ideas_by_phase=ideas_by_phase,
+                    context=context
+                )
+                
+                # Add cost of conclusion synthesis if available
+                if hasattr(result.conclusion, 'metadata') and 'llm_cost' in result.conclusion.metadata:
+                    result.llm_cost += result.conclusion.metadata['llm_cost']
+                    
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)